client/src/templates/Challenges/classic/show.tsx

```diff
@@ -150,7 +146,22 @@ const handleContentWidgetEvents = (e: MouseEvent | TouchEvent): void => {
   }
 };
 
-// Component
+const StepPreview = ({
+  disableIframe
+}: Pick<PreviewProps, 'disableIframe'>) => {
+  return (
+    <Preview
+      className='full-height'
+      disableIframe={disableIframe}
+      previewMounted={previewMounted}
+    />
+  );
+};
+
+const defaultOutput = `/**
+* Your test output will go here
+*/`;
```

Comment:
```suggestion
const defaultOutput = `
/**
* Your test output will go here
*/`;
```
The newline is important, because this text ends up in a `pre` block.